# Korean translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# darehanl <darehanl@gmail.com>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2013-05-22 21:30+0100\n"
"PO-Revision-Date: 2014-03-16 18:07+0000\n"
"Last-Translator: Kim Boram <boramism@gmail.com>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Launchpad-Export-Date: 2014-04-23 06:52+0000\n"
"X-Generator: Launchpad (build 16985)\n"
"Language: ko\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:211 ../DistUpgrade/distro.py:441
#, python-format
msgid "Server for %s"
msgstr "%s 서버"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:229 ../DistUpgrade/distro.py:235
#: ../DistUpgrade/distro.py:251
msgid "Main server"
msgstr "주 서버"

#: ../DistUpgrade/distro.py:255
msgid "Custom servers"
msgstr "사용자 정의 서버"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:142
msgid "Could not calculate sources.list entry"
msgstr "sources.list 항목을 계산할 수 없습니다."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:251
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr "어떤 패키지 파일도 찾을 수 없습니다. 우분투 디스크가 아니거나 올바르지 않은 아키텍처인 것은 아닙니까?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:294
msgid "Failed to add the CD"
msgstr "CD 추가 실패"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:295
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"CD를 추가할 때 오류가 발생하여 업그레이드를 중단합니다. 올바른 우분투 CD를 사용했다면 이 버그를 보고해 주십시오.\n"
"\n"
"오류 메시지:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:152
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "오류가 있는 패키지 제거"

#: ../DistUpgrade/DistUpgradeCache.py:155
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"'%s' 패키지는 불완전한 상태이며 다시 설치해야 하지만, 저장소에서 찾을 수 없습니다. 이 패키지를 제거한 후 계속하시겠습니까?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:256
msgid "The server may be overloaded"
msgstr "서버가 과부화 상태인 것 같습니다"

#: ../DistUpgrade/DistUpgradeCache.py:369
msgid "Broken packages"
msgstr "망가진 패키지"

#: ../DistUpgrade/DistUpgradeCache.py:370
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr "이 소프트웨어로 고칠 수 없는 망가진 패키지가 있습니다. 진행하기 전에 시냅틱이나 apt-get 명령으로 복구하십시오."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:693
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"업데이트를 계산하는 중 해결할 수 없는 문제가 생겼습니다.:\n"
"%s\n"
"\n"
" 이것의 원인은 다음 중 하나일 수 있습니다:\n"
" * 차기 버전의 우분투 업그레이드로 업그레이드하는 중이거나\n"
" * 차기 버전의 우분투를 사용하고 있거나\n"
" * 써드 파티 소프트웨어를 사용 중입니다.\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:648
msgid "This is most likely a transient problem, please try again later."
msgstr "이것은 아마도 일시적인 문제일 것입니다. 나중에 다시 시도해주십시오."

#: ../DistUpgrade/DistUpgradeCache.py:651
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"이 중 어떤 것도 적용할 수 없는 경우에는 터미널에서 'ubuntu-bug ubuntu-release-upgrader-core' 명령을 "
"사용해 버그를 보고해주십시오."

#: ../DistUpgrade/DistUpgradeCache.py:656
msgid "Could not calculate the upgrade"
msgstr "업그레이드를 계산할 수 없습니다."

#: ../DistUpgrade/DistUpgradeCache.py:707
msgid "Error authenticating some packages"
msgstr "일부 패키지를 인증할 수 없습니다."

#: ../DistUpgrade/DistUpgradeCache.py:708
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"몇몇 패키지를 인증할 수 없습니다. 일시적인 네트워크 문제일 수 있으므로 나중에 대시 시도해주십시오. 인증하지 못한 패키지의 목록은 "
"다음과 같습니다."

#: ../DistUpgrade/DistUpgradeCache.py:728
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr "패키지 '%s'은(는) 제거 차단 목록에 기록되어 있어 제거할 수 없습니다."

#: ../DistUpgrade/DistUpgradeCache.py:732
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "필수 패키지 '%s'을(를) 제거할 항목으로 표시했습니다."

#: ../DistUpgrade/DistUpgradeCache.py:741
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "차단 목록 상의 버전 '%s'을(를) 설치합니다"

#: ../DistUpgrade/DistUpgradeCache.py:859
#, python-format
msgid "Can't install '%s'"
msgstr "'%s'을(를) 설치할 수 없음"

#: ../DistUpgrade/DistUpgradeCache.py:860
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"필요한 패키지를 설치할 수 없습니다. 터미널에서 'ubuntu-bug ubuntu-release-upgrader-core' 명령을 사용해 "
"버그를 보고해주십시오."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:871
msgid "Can't guess meta-package"
msgstr "메타 패키지를 추측할 수 없음"

#: ../DistUpgrade/DistUpgradeCache.py:872
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"시스템에 ubuntu-desktop이나 kubuntu-desktop이나 xubuntu-desktop 또는 edubuntu-desktop "
"패키지가 없으며, 현재 실행 중인 우분투의 버전을 알아낼 수 없습니다.\n"
"우선 위의 패키지 중 하나를 시냅틱이나 apt-get 명령으로 설치하시기 바랍니다."

#: ../DistUpgrade/DistUpgradeController.py:126
msgid "Reading cache"
msgstr "캐시 읽는 중"

#: ../DistUpgrade/DistUpgradeController.py:230
msgid "Unable to get exclusive lock"
msgstr "배타적으로 잠글 수 없음"

#: ../DistUpgrade/DistUpgradeController.py:231
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"이것은 보통 또 다른 패키지 관리 프로그램(예를 들어 apt-get이나 aptitude)을 이미 실행하고 있는 것을 의미 합니다. 우선 "
"해덩 프로그램을 종료하십시오."

#: ../DistUpgrade/DistUpgradeController.py:289
msgid "Upgrading over remote connection not supported"
msgstr "원격 접속을 통한 업그레이드를 지원하지 않음"

#: ../DistUpgrade/DistUpgradeController.py:290
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"원격 SSH접속을 통한 업그레이드를 지원하지 않는 SS H프론트엔드를 사용하여 업그레이드를 실행하고 있습니다. 'do-release-"
"upgrade' 명령으로 텍스트모드 업그레이드를 사요해주십시오.\n"
"\n"
"업그레이드를 중지될 것입니다. SSH를 사용하지 않고 다시 시도하십시오."

#: ../DistUpgrade/DistUpgradeController.py:304
msgid "Continue running under SSH?"
msgstr "SSH를 통해 계속 진행하시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:305
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"현재 세션은 SSH에서 돌아가고 있습니다. SSH를 이용한 세션에서 업그레이드를 하는 것은 추천하지 않습니다. 업그레이드에 실패할 경우 "
"복구하기가 힘들어 집니다.\n"
"\n"
"진행하시면 '%s' 포트로 추가 SSH데몬을 시작할 것입니다.\n"
"계속 진행하시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:319
msgid "Starting additional sshd"
msgstr "추가 sshd 시작"

#: ../DistUpgrade/DistUpgradeController.py:320
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"문제가 발생했을 때 복구를 쉽게 할 수 있도록 sshd 데몬을 포트 '%s'에 추가로 실행합니다. 현재 실행 중인 ssh에 문제가 "
"발생해도 추가로 실행한 데몬을 통해 접속할 수 있습니다.\n"

#: ../DistUpgrade/DistUpgradeController.py:328
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"방화벽을 사용 중이시라면 임시로 이 포트를 열어 주셔야 합니다. 포트를 여는 것은 잠재적으로 위험하기 때문에 자동으로 실행되지 않습니다. "
"아래와 같이 포트를 여실 수 있습니다 : '%s'"

#: ../DistUpgrade/DistUpgradeController.py:400
#: ../DistUpgrade/DistUpgradeController.py:445
msgid "Can not upgrade"
msgstr "업그레이드할 수 없음"

#: ../DistUpgrade/DistUpgradeController.py:401
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "이 도구로  '%s'에서 '%s'(으)로 업그레이드할 수 없습니다."

#: ../DistUpgrade/DistUpgradeController.py:410
msgid "Sandbox setup failed"
msgstr "샌드박스 설정 실패"

#: ../DistUpgrade/DistUpgradeController.py:411
msgid "It was not possible to create the sandbox environment."
msgstr "샌드박스 환경을 만들 수 없습니다."

#: ../DistUpgrade/DistUpgradeController.py:417
msgid "Sandbox mode"
msgstr "샌드박스 모드"

#: ../DistUpgrade/DistUpgradeController.py:418
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""
"이 업그레이드는 샌드 박스(테스트) 모드에서 실행하고 있습니다. 모든 바뀐 내용은 '%s'에 기록하며 다시 시작하게 되면 유실됩니다.\n"
"\n"
"지금부터 다음 다시 시작할 때까지 어떤 바뀐 내용도 시스템 디렉터리에 기록하지 않습니다."

#: ../DistUpgrade/DistUpgradeController.py:446
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr "파이썬 설치가 잘못되었습니다. '/usr/bin/python' 심볼릭 링크를 수정하십시오."

#: ../DistUpgrade/DistUpgradeController.py:472
msgid "Package 'debsig-verify' is installed"
msgstr "'debsig-verify' 패키지를 설치했습니다."

#: ../DistUpgrade/DistUpgradeController.py:473
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"해당 패키지를 설치한 상태로는 업그레이드를 계속할 수 없습니다.\n"
"시냅틱이나 'apt-get remove debsig-verify' 명령으로 제거한 후 다시 업그레이드를 수행하십시오."

#: ../DistUpgrade/DistUpgradeController.py:485
#, python-format
msgid "Can not write to '%s'"
msgstr "'%s'에 기록할 수 없습니다"

#: ../DistUpgrade/DistUpgradeController.py:486
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"사용자 시스템의 시스템 디렉터리 '%s'에 기록할 수 없습니다. 업그레이드를 계속할 수 없습니다.\n"
"시스템 디렉터리에 기록할 수 있는 권한이 있는지 확인하십시오."

#: ../DistUpgrade/DistUpgradeController.py:497
msgid "Include latest updates from the Internet?"
msgstr "인터넷으로 최신 업데이트를 설치하시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:498
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"업그레이드 프로그램은 업그레이드 중 인터넷을 이용해 자동으로 최신 업데이트를 다운로드해 설치할 수 있습니다. 인터넷에 연결되어 있다면 "
"가급적 설치하는 것이 좋습니다.\n"
"\n"
"업그레이드는 더 오래 걸리겠지만 완료 후에는 시스템이 완전히 최신 상태가 됩니다. 지금 설치를 하지 않기로 했다면 업그레이드를 마친 다음 "
"즉시 최신 업데이트를 설치해야 합니다.\n"
"'아니오'를 선택하면 네트워크를 이용하지 않습니다."

#: ../DistUpgrade/DistUpgradeController.py:729
#, python-format
msgid "disabled on upgrade to %s"
msgstr "%s(으)로 업그레이드 하지 않음"

#: ../DistUpgrade/DistUpgradeController.py:762
msgid "No valid mirror found"
msgstr "올바른 미러 서버를 찾지 못함"

#: ../DistUpgrade/DistUpgradeController.py:753
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"저장소 정보를 찾아봤지만 업그레이드를 위한 미러 항목을 찾을 수 없습니다. 내부 미러를 운영 중이거나 혹은 미러 정보가 최신이 아닐 수 "
"있습니다.\n"
"\n"
"그래도 'sources.list' 파일을 다시 작성 하시겠습니까? 여기서 '예'를 선택하면 모든 '%s' 항목을 '%s'(으)로  "
"업데이트합니다.\n"
"'아니오'를 선택하면 업그레이드를 취소합니다."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:783
msgid "Generate default sources?"
msgstr "기본 소스 목록을 만드시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:784
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"'sources.list' 파일을 찾아봤지만 '%s'에 대한 항목을 찾을 수 없습니다.\n"
"'%s'에 대한 기본 항목을 추가하시겠습니까? '아니오'를 선택하면 업그레이드를 취소합니다."

#: ../DistUpgrade/DistUpgradeController.py:819
msgid "Repository information invalid"
msgstr "저장소 정보가 올바르지 않습니다"

#: ../DistUpgrade/DistUpgradeController.py:820
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr "저장소 정보를 업데이트한 결과 올바르지 않은 파일이 생성되었습니다. 버그 보고 작업을 시작합니다."

#: ../DistUpgrade/DistUpgradeController.py:827
msgid "Third party sources disabled"
msgstr "서드 파티 소스는 사용할 수 없습니다"

#: ../DistUpgrade/DistUpgradeController.py:828
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"souces.list에서 서드 파티 목록의 일부를 이용할 수 없습니다. 업그레이드를 마친 후 '소프트웨어 소스' 도구나 패키지 관리자를 "
"이용해 다시 사용 할 수 있습니다."

#: ../DistUpgrade/DistUpgradeController.py:870
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "패키지 상태가 불완전함"

#: ../DistUpgrade/DistUpgradeController.py:873
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"'%s' 패키지는 불완전한 상태이며 다시 설치해야합니다만, 저장소에서 찾을 수 없습니다.  이 패키지를 직접 다시 설치하거나 시스템에서 "
"제거하십시오."

#: ../DistUpgrade/DistUpgradeController.py:921
msgid "Error during update"
msgstr "업데이트 중 오류 발생"

#: ../DistUpgrade/DistUpgradeController.py:922
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"업데이트 중 문제가 발생했습니다. 보통 네트워크 문제인 경우가 많습니다.네트워크의 연결 상태를 확인하시고 다시 시도하십시오."

#: ../DistUpgrade/DistUpgradeController.py:931
msgid "Not enough free disk space"
msgstr "디스크 여유 공간이 충분하지 않습니다."

#: ../DistUpgrade/DistUpgradeController.py:932
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"업그레이드를 중단했습니다. 이 업그레이드는 %s의 용량이 드라이브 '%s'에 필요합니다.최소한 %s의 용량을 '%s'에 확보해주십시오. "
"휴지통을 비워주시고 'sudo apt-get clean' 명령으로 이전에 설치하며 만들어진 임시 패키지를 제거하십시오."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:961
#: ../DistUpgrade/DistUpgradeController.py:1743
msgid "Calculating the changes"
msgstr "바뀐 내용을 계산하는 중"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:993
msgid "Do you want to start the upgrade?"
msgstr "업그레이드를 시작하시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:1066
msgid "Upgrade canceled"
msgstr "업그레이드 취소함"

#: ../DistUpgrade/DistUpgradeController.py:1067
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr "업그레이드를 취소합니다. 시스템은 업그레이드 이전 상태로 돌아가며 업그레이드는 이후에도 언제든지 다시 할 수 있습니다."

#: ../DistUpgrade/DistUpgradeController.py:1073
#: ../DistUpgrade/DistUpgradeController.py:1197
msgid "Could not download the upgrades"
msgstr "업그레이드 파일을 다운로드 할 수 없음"

#: ../DistUpgrade/DistUpgradeController.py:1074
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr "업그레이드를 중단했습니다. 인터넷 연결이나 설치 매체를 확인한 후 다시 시도하십시오. 다운로드한 파일은 계속 보존합니다."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1148
#: ../DistUpgrade/DistUpgradeController.py:1185
#: ../DistUpgrade/DistUpgradeController.py:1290
msgid "Error during commit"
msgstr "커밋 작업 중 오류 발생"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1150
#: ../DistUpgrade/DistUpgradeController.py:1187
#: ../DistUpgrade/DistUpgradeController.py:1329
msgid "Restoring original system state"
msgstr "시스템을 이전의 상태로 복구하고 있습니다"

#: ../DistUpgrade/DistUpgradeController.py:1151
#: ../DistUpgrade/DistUpgradeController.py:1166
#: ../DistUpgrade/DistUpgradeController.py:1188
msgid "Could not install the upgrades"
msgstr "업그레이드를 설치하지 못했습니다."

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1156
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"업그레이드를 중단했습니다. 시스템에 치명적인 오류가 발생했을 수 있습니다.복구(dpkg --configure -a) 명령을 실행하겠습니다."

#: ../DistUpgrade/DistUpgradeController.py:1161
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at "
"http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug "
"and attach the files in /var/log/dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"브라우저에서 http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-"
"upgrader/+filebug and attach the files in /var/log/dist-upgrade/ 페이지를 방문해 "
"버그를 보고해주십시오.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1198
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr "업그레이드를 중단했습니다. 인터넷 연결이나 설치 매체를 확인한 후 다시 시도하십시오. "

#: ../DistUpgrade/DistUpgradeController.py:1278
msgid "Remove obsolete packages?"
msgstr "사용하지 못하게 된 패키지를 제거하시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:1279
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "유지(_K)"

#: ../DistUpgrade/DistUpgradeController.py:1279
msgid "_Remove"
msgstr "제거(_R)"

#: ../DistUpgrade/DistUpgradeController.py:1291
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr "정리하는 도중에 문제가 발생하였습니다. 다음 메시지를 통해 더 많은 정보를 확인할 수 있습니다. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1367
msgid "Required depends is not installed"
msgstr "필요한 의존 프로그램을 설치하지 않았습니다."

#: ../DistUpgrade/DistUpgradeController.py:1368
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "필요한 의존 프로그램 '%s'을(를) 설치하지 않습니다. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1636
#: ../DistUpgrade/DistUpgradeController.py:1701
msgid "Checking package manager"
msgstr "패키지 관리자 확인 중"

#: ../DistUpgrade/DistUpgradeController.py:1641
msgid "Preparing the upgrade failed"
msgstr "업그레이드 준비에 실패했습니다"

#: ../DistUpgrade/DistUpgradeController.py:1642
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr "시스템 업그레이드 준비를 실패했습니다. 버그 보고 작업을 시작합니다."

#: ../DistUpgrade/DistUpgradeController.py:1656
msgid "Getting upgrade prerequisites failed"
msgstr "업그레이드 사전 작업에 실패했습니다"

#: ../DistUpgrade/DistUpgradeController.py:1657
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"시스템에서 업그레이드 사전 작업을 할 수 없습니다. 업그레이드를 취소한 후 이전 시스템 상태로 되돌립니다.\n"
"\n"
"추가적으로, 버그 보고 작업을 시작합니다."

#: ../DistUpgrade/DistUpgradeController.py:1685
msgid "Updating repository information"
msgstr "저장소 정보 업데이트 중"

#: ../DistUpgrade/DistUpgradeController.py:1692
msgid "Failed to add the cdrom"
msgstr "cdrom을 추가할 수 없음"

#: ../DistUpgrade/DistUpgradeController.py:1693
msgid "Sorry, adding the cdrom was not successful."
msgstr "죄송합니다. cdrom을 추사할 수 없습니다."

#: ../DistUpgrade/DistUpgradeController.py:1724
msgid "Invalid package information"
msgstr "올바르지 않은 패키지 정보"

#: ../DistUpgrade/DistUpgradeController.py:1725
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"패키지 정보를 업데이트한 후 핵심 패키지'%s'을(를) 찾을 수 없습니다..소프트웨어에 공식 미러가 없거나 사용 중인 미러에 과부하로 "
"인해 발생할 수 있습니다. /etc/apt/sources.list 파일을 통해 현재 목록에 설정된 소스프ㅌ웨어 소스를 확인할 수 "
"있습니다.\n"
"과도한 부하로 인해 발생한 경우에는 잠시 후 다시 업그레이드를  시작해주십시오."

#: ../DistUpgrade/DistUpgradeController.py:1749
#: ../DistUpgrade/DistUpgradeController.py:1805
msgid "Fetching"
msgstr "가져오는 중"

#: ../DistUpgrade/DistUpgradeController.py:1755
#: ../DistUpgrade/DistUpgradeController.py:1809
msgid "Upgrading"
msgstr "업그레이드 중"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1760
#: ../DistUpgrade/DistUpgradeController.py:1811
#: ../DistUpgrade/DistUpgradeController.py:1818
#: ../DistUpgrade/DistUpgradeController.py:1829
msgid "Upgrade complete"
msgstr "업그레이드 완료"

#: ../DistUpgrade/DistUpgradeController.py:1761
#: ../DistUpgrade/DistUpgradeController.py:1812
#: ../DistUpgrade/DistUpgradeController.py:1819
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "업그레이드를 완료했지만 업그레이드 과정 중 오류가 발생하였습니다."

#: ../DistUpgrade/DistUpgradeController.py:1768
msgid "Searching for obsolete software"
msgstr "못 쓰게 된 소프트웨어를 검색하는 중"

#: ../DistUpgrade/DistUpgradeController.py:1781
msgid "System upgrade is complete."
msgstr "시스템 업그레이드를 완료했습니다."

#: ../DistUpgrade/DistUpgradeController.py:1830
msgid "The partial upgrade was completed."
msgstr "부분 업그레이드를 완료했습니다."

#: ../DistUpgrade/DistUpgradeFetcher.py:123
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:119
msgid "Could not find the release notes"
msgstr "릴리즈 정보를 찾을 수 없습니다."

#: ../DistUpgrade/DistUpgradeFetcher.py:124
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:120
msgid "The server may be overloaded. "
msgstr "서버에 접속자가 너무 많습니다. "

#: ../DistUpgrade/DistUpgradeFetcher.py:136
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:124
msgid "Could not download the release notes"
msgstr "릴리즈 정보를 다운로드 할 수 없습니다."

#: ../DistUpgrade/DistUpgradeFetcher.py:137
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:125
msgid "Please check your internet connection."
msgstr "인터넷 연결 상태를 확인하십시오."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:75
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "'%(signature)s'(으)로 '%(file)s' 파일 인증 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:136
#, python-format
msgid "extracting '%s'"
msgstr "'%s' 압축 해제 중"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:157
#: ../DistUpgrade/DistUpgradeFetcherCore.py:158
msgid "Could not run the upgrade tool"
msgstr "업그레이드 도구를 실행할 수 없습니다."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:159
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"이 것은 업그레이드 도구의 버그일 가능성이 높습니다. 'ubuntu-bug ubuntu-release-upgrader-core' 명령을 "
"이용해 버그를 보고해주십시오."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:236
msgid "Upgrade tool signature"
msgstr "업그레이드 도구 서명"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Upgrade tool"
msgstr "업그레이드 도구"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
msgid "Failed to fetch"
msgstr "가져오기 실패"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:278
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "업그레이드를 가져올 수 없습니다. 네트워크에 문제가 있을 수 있습니다. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:282
msgid "Authentication failed"
msgstr "인증 실패"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:283
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr "업그레이드를 인증할 수 없습니다.네트워크나 서버에 문제가 있을 수 있습니다. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:288
msgid "Failed to extract"
msgstr "압축 해체 실패"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:289
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr "업그레이드의 압축을 해체할 수 없습니다. 네트워크나 서버에 문제가 있을 수 있습니다. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:294
msgid "Verification failed"
msgstr "검증 실패"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:295
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr "업그레이드를 검증할 수 없습니다.  네트워크나 서버에 문제가 있을 수 있습니다. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:309
#: ../DistUpgrade/DistUpgradeFetcherCore.py:315
msgid "Can not run the upgrade"
msgstr "업그레이드를 할 수 없습니다."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:310
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"이 것은 보통 /tmp 디렉터리를 noexec로 마운트한 시스템에서 발생합니다. noexec 없이 다시 마운트한 후 업그레이드를 "
"시작하십시오."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:316
#, python-format
msgid "The error message is '%s'."
msgstr "오류 메시지는 '%s'입니다."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:70
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:101
msgid "Upgrade"
msgstr "업그레이드"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:105
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "릴리즈 정보"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:147
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:160
msgid "Downloading additional package files..."
msgstr "추가 패키지 파일 다운로드 중..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:162
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "파일 %s / %s, 속도 %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:166
#, python-format
msgid "File %s of %s"
msgstr "파일 %s / %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:173
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:115
#: ../DistUpgrade/DistUpgradeViewKDE.py:196
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "'%s'(을)를 '%s' 드라이브에 넣어주십시오."

#. change = QMessageBox.question(None, _("Media Change"), msg,
#. QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:177
#: ../DistUpgrade/DistUpgradeViewKDE.py:197
msgid "Media Change"
msgstr "미디어 바꾸기"

#: ../DistUpgrade/DistUpgradeQuirks.py:208
msgid "evms in use"
msgstr "evms 사용 중"

#: ../DistUpgrade/DistUpgradeQuirks.py:209
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"이 시스템은 /proc/mounts 안에 'evms' 볼륨 매니저를 사용하고 있습니다.  'evms'은 더 이상 지원되지 않으므로, 종료 "
"후 업그레이드를 다시 실행하십시오."

#: ../DistUpgrade/DistUpgradeQuirks.py:511
msgid "Your graphics hardware may not be fully supported in Ubuntu 13.04."
msgstr "사용자의 그래픽 하드웨어가 우분투 13.04를 완벽하게 지원하지 않을 수 있습니다."

#: ../DistUpgrade/DistUpgradeQuirks.py:248
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D Do you still "
"want to continue with the upgrade?"
msgstr ""
"그래픽 하드웨어가 '유니티' 데스크톱 환경을 완벽하게 지원하지 않습니다. 업그레이드 후 매우 느린 환경을 이용하게 될 수 있으니 LTS "
"버전을 이용할 것을 추천합니다. 더 많은 정보는 "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D 페이지를 확인하십시오. "
"업그레이드를 계속하시겠습니까?"

#: ../DistUpgrade/DistUpgradeQuirks.py:272
msgid ""
"Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr "그래픽 카드가 우분투 12.04 장기 지원판을 지원하지 않습니다."

#: ../DistUpgrade/DistUpgradeQuirks.py:274
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"인텔 그래픽 카드가 우분투 12.04 장기 지원판을 완전하게 지원하지 않아 업그레이드 후 문제가 발생할 수 있습니다. 더 자세한 내용은 "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx 페이지를 확인해주십시오. "
"업그레이드를 계속 진행하시겠습니까?"

#: ../DistUpgrade/DistUpgradeQuirks.py:294
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr "업그레이드로 인해 데스크탑 효과나 게임, 그래픽 관련 프로그램의 성능이 줄어들 수 있습니다."

#: ../DistUpgrade/DistUpgradeQuirks.py:564
#: ../DistUpgrade/DistUpgradeQuirks.py:592
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"이 컴퓨터는 현재 NVIDIA사의 'nvidia' 그래픽 드라이버를 사용하고 있습니다. 이 드라이버는 우분투 10.04 LTS을 지원하는 "
"버전이 없습니다.\n"
"계속 진행하시겠습니까?"

#: ../DistUpgrade/DistUpgradeQuirks.py:298
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"이 컴퓨터는 현재 AMD사의 'fglrx' 그래픽 드라이버를 사용하고 있습니다. 이 드라이버는 우분투 10.04 LTS을 지원하는 버전이 "
"없습니다.\n"
"계속 진행하시겠습니까?"

#: ../DistUpgrade/DistUpgradeQuirks.py:333
msgid "No i686 CPU"
msgstr "i686 CPU 아님"

#: ../DistUpgrade/DistUpgradeQuirks.py:334
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"현 시스템은 i586 호환 CPU가 아니거나, 'cmov' 기능이 없는 CPU를 사용하고 있습니다. 모든 패키지는 최소 i686 호환 "
"CPU가 필요합니다. 현 시스템에서 새로운 우분투 버전으로 업그레이드 할 수 없습니다."

#: ../DistUpgrade/DistUpgradeQuirks.py:370
msgid "No ARMv6 CPU"
msgstr "ARMv6 CPU 아님"

#: ../DistUpgrade/DistUpgradeQuirks.py:371
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"시스템이 ARMv6 보다 더 오래된 아키텍처의 ARM CPU를 사용하고 있습니다. 우분투 karmic의 모든 꾸러미는 ARMv6에 최적화 "
"되어있기 때문에 최소 CPU로 ARMv6이 필요합니다. 현재 하드웨어로는 새로운 우분투 버전으로 시스템을 업그레이드 할 수 없습니다."

#: ../DistUpgrade/DistUpgradeQuirks.py:391
msgid "No init available"
msgstr "init 사용 불가"

#: ../DistUpgrade/DistUpgradeQuirks.py:392
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"현재 시스템은 init 데몬(예:Linux-VServer)가 없는 가상 환경인 것으로 감지되었습니다. 우분투 10.04 LTS는 이러한 "
"환경에서 제대로 동작할 수 없으며 우선 가상 환경의 설정을 조정해야합니다.\n"
"\n"
"계속 진행하시겠습니까?"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid "Sandbox upgrade using aufs"
msgstr "aufs로 샌드박스 업그레이드"

#: ../DistUpgrade/DistUpgradeMain.py:66
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "주어진 경로로 업그레이드할 수 있는 패키지가 있는 시디롬을 검색합니다."

#: ../DistUpgrade/DistUpgradeMain.py:72
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"프론트 엔드를 사용합니다. 현재 사용할 수 있는 것은 다음과 같습니다: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "*DEPRECATED* this option will be ignored"
msgstr "*삭제된 옵션* 이 항목은 무시합니다."

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "부분 업그레이드만 실행합니다.(sources.list 파일은 바꾸지 않음)"

#: ../DistUpgrade/DistUpgradeMain.py:81
msgid "Disable GNU screen support"
msgstr "GNU 화면 지원 사용하지 않음"

#: ../DistUpgrade/DistUpgradeMain.py:83
msgid "Set datadir"
msgstr "datadir 설정"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:136
#: ../DistUpgrade/DistUpgradeViewKDE.py:210
msgid "Fetching is complete"
msgstr "가져오기 완료"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk3.py:151
#: ../DistUpgrade/DistUpgradeViewKDE.py:226
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "%li번째 파일(전체 %li개)을 %sB/s의 속로로 가져오는 중"

#: ../DistUpgrade/DistUpgradeViewGtk.py:151
#: ../DistUpgrade/DistUpgradeViewGtk.py:298
#: ../DistUpgrade/DistUpgradeViewGtk3.py:153
#: ../DistUpgrade/DistUpgradeViewGtk3.py:310
#: ../DistUpgrade/DistUpgradeViewKDE.py:227
#: ../DistUpgrade/DistUpgradeViewKDE.py:375
#, python-format
msgid "About %s remaining"
msgstr "약 %s 남음"

#: ../DistUpgrade/DistUpgradeViewGtk.py:154
#: ../DistUpgrade/DistUpgradeViewGtk3.py:156
#: ../DistUpgrade/DistUpgradeViewKDE.py:229
#, python-format
msgid "Fetching file %li of %li"
msgstr "%li번째 파일(전체 %li개)을 가져오는 중"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:185
#: ../DistUpgrade/DistUpgradeViewGtk3.py:187
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "Applying changes"
msgstr "바뀐 내용을 적용하는 중"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:210
#: ../DistUpgrade/DistUpgradeViewGtk3.py:213
#: ../DistUpgrade/DistUpgradeViewKDE.py:279
msgid "dependency problems - leaving unconfigured"
msgstr "의존성 오류 - 설정하지 않은 채로 남겨둡니다."

#: ../DistUpgrade/DistUpgradeViewGtk.py:215
#: ../DistUpgrade/DistUpgradeViewGtk3.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:281
#, python-format
msgid "Could not install '%s'"
msgstr "'%s'(을)를 설치할 수 없음"

#: ../DistUpgrade/DistUpgradeViewGtk.py:216
#: ../DistUpgrade/DistUpgradeViewGtk3.py:219
#: ../DistUpgrade/DistUpgradeViewKDE.py:282
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr "업그레이드는 계속 진행하지만 '%s' 패키지가 동작하지 않을 수도 있습니다. 이 버그를 보고해주십시오."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:233
#: ../DistUpgrade/DistUpgradeViewGtk3.py:236
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"사용자가 작접 설정한 설정 파일 '%s'을(를)\n"
"바꾸시겠습니까?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:234
#: ../DistUpgrade/DistUpgradeViewGtk3.py:237
#: ../DistUpgrade/DistUpgradeViewKDE.py:304
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr "새로운 버전으로 바꾸기를 선택하면 이전 설정 파일의 바뀐 내용을 잃게 됩니다."

#: ../DistUpgrade/DistUpgradeViewGtk.py:253
#: ../DistUpgrade/DistUpgradeViewGtk3.py:257
#: ../DistUpgrade/DistUpgradeViewKDE.py:327
msgid "The 'diff' command was not found"
msgstr "'diff' 명령을 찾을 수 없습니다"

#: ../DistUpgrade/DistUpgradeViewGtk.py:466
#: ../DistUpgrade/DistUpgradeViewGtk3.py:478
#: ../DistUpgrade/DistUpgradeViewText.py:100
msgid "A fatal error occurred"
msgstr "치명적인 오류 발생"

#: ../DistUpgrade/DistUpgradeViewGtk.py:467
#: ../DistUpgrade/DistUpgradeViewGtk3.py:479
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"이 버그를 /var/log/dist-upgrade/main.log와 /var/log/dist-upgrade/apt.log를 첨부하여 "
"보고해주십시오. 업그레이드를 중단했습니다.\n"
"기존 sources.list 파일은 /etc/apt/sources.list.distUpgrade 파일로 저장했습니다."

#: ../DistUpgrade/DistUpgradeViewGtk.py:484
#: ../DistUpgrade/DistUpgradeViewGtk3.py:496
msgid "Ctrl-c pressed"
msgstr "컨트롤-C 누름"

#: ../DistUpgrade/DistUpgradeViewGtk.py:485
#: ../DistUpgrade/DistUpgradeViewGtk3.py:497
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr "이 작업을 취소하면 시스템을 사용할 수 없게 될 수도 있습니다. 정말 취소하시겠습니까?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:633
#: ../DistUpgrade/DistUpgradeViewGtk3.py:630
msgid "To prevent data loss close all open applications and documents."
msgstr "데이터 손실을 막으려면 열려있는 모든 프로그램과 문서를 닫으십시오."

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:644
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "캐노니컬에서 더 이상 지원하지 않습니다 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:645
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>다운그레이드 (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:646
#, python-format
msgid "Remove (%s)"
msgstr "지우기 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:650
#: ../DistUpgrade/DistUpgradeViewGtk3.py:647
#, python-format
msgid "No longer needed (%s)"
msgstr "더 이상 필요하지 않음 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:651
#: ../DistUpgrade/DistUpgradeViewGtk3.py:648
#, python-format
msgid "Install (%s)"
msgstr "설치 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:652
#: ../DistUpgrade/DistUpgradeViewGtk3.py:649
#, python-format
msgid "Upgrade (%s)"
msgstr "업그레이드(%s)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:339
msgid "Show Difference >>>"
msgstr "차이점 보이기 >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:342
msgid "<<< Hide Difference"
msgstr "<<< 차이점 숨기기"

#: ../DistUpgrade/DistUpgradeViewKDE.py:558
msgid "Error"
msgstr "오류"

#: ../DistUpgrade/DistUpgradeViewKDE.py:572
msgid "&Cancel"
msgstr "취소 (&C)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:817
msgid "&Close"
msgstr "닫기(&C)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:622
msgid "Show Terminal >>>"
msgstr "터미널 보이기 >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:625
msgid "<<< Hide Terminal"
msgstr "<<< 터미널 숨기기"

#: ../DistUpgrade/DistUpgradeViewKDE.py:705
msgid "Information"
msgstr "정보"

#: ../DistUpgrade/DistUpgradeViewKDE.py:755
#: ../DistUpgrade/DistUpgradeViewKDE.py:800
#: ../DistUpgrade/DistUpgradeViewKDE.py:803
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "자세한 내용"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#, python-format
msgid "No longer supported %s"
msgstr "지원하지 않음 %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:783
#, python-format
msgid "Remove %s"
msgstr "제거  %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#: ../DistUpgrade/DistUpgradeViewText.py:194
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "제거 (자동으로 설치함) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:787
#, python-format
msgid "Install %s"
msgstr "설치 %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:789
#, python-format
msgid "Upgrade %s"
msgstr "업그레이드 %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:813
#: ../DistUpgrade/DistUpgradeViewText.py:242
msgid "Restart required"
msgstr "다시 시작해야 합니다."

#: ../DistUpgrade/DistUpgradeViewKDE.py:813
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>업그레이드를 완료하기 위해 시스템을 다시 시작합니다.</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:816
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "지금 다시 시작(_R)"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:834
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>진행 중인 업그레이드를 취소하시겠습니까?</big></b>\n"
"\n"
"업그레이드 중간에 취소하면 시스템을 사용할 수 없게 될 수 있습니다. 업그레이드를 계속 진행할 것을 강력하게 추천합니다."

#: ../DistUpgrade/DistUpgradeViewKDE.py:838
msgid "Cancel Upgrade?"
msgstr "업그레이드를 취소하겠습니까?"

#: ../DistUpgrade/DistUpgradeView.py:73
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li 일"

#: ../DistUpgrade/DistUpgradeView.py:75
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li 시간"

#: ../DistUpgrade/DistUpgradeView.py:77
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li 분"

#: ../DistUpgrade/DistUpgradeView.py:78
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li 초"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:94
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:112
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:163
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr "이 다운로드는 1Mbit DSL 연결로는 약 %s, 56k 모뎀으로는 약 %s 정도 가 필요합니다."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:167
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "현재 연결 상태로는 다운로드 과정에 약 %s 정도가 필요합니다. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:271
#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "업그레이드를 준비하는 중"

#: ../DistUpgrade/DistUpgradeView.py:272
msgid "Getting new software channels"
msgstr "새 소프트웨어 채널을 가져오는 중"

#: ../DistUpgrade/DistUpgradeView.py:273
#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "새 패키지를 가져오는 중"

#: ../DistUpgrade/DistUpgradeView.py:274
#: ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "업그레이드 설치하는 중"

#: ../DistUpgrade/DistUpgradeView.py:275
#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "마무리 중"

#: ../DistUpgrade/DistUpgradeView.py:360
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] "설치한 패키지 중 %(amount)d개는 더 이상 캐노니컬이 지원하지 않습니다.하지만 공동체는 계속 지원합니다."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:369
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "패키지 %d개를 제거할 것입니다."

#: ../DistUpgrade/DistUpgradeView.py:374
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "새 패키지 %d개를 설치할 것입니다."

#: ../DistUpgrade/DistUpgradeView.py:380
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "패키지 %d개를 업그레이드 할 것입니다."

#: ../DistUpgrade/DistUpgradeView.py:388
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"모두 %s개의 패키지를 다운로드해야 합니다.. "

#: ../DistUpgrade/DistUpgradeView.py:393
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr "업그레이드를 다운로드하고 설치하는 것은 긴 시간이 필요할 수도 있으며, 한번 다운로드가 끝나면 취소할 수 없습니다."

#: ../DistUpgrade/DistUpgradeView.py:397
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr "업그레이드를 다운로드하고 설치하는 것은 수 시간이 필요할 수도 있으며, 한번 다운로드가 끝나면 취소할 수 없습니다."

#: ../DistUpgrade/DistUpgradeView.py:402
msgid "Removing the packages can take several hours. "
msgstr "패키지 제거는 수 시간이 걸릴 수 있습니다. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:407
msgid "The software on this computer is up to date."
msgstr "이 컴퓨터의 소프트웨어를 모두 업데이트했습니다."

#: ../DistUpgrade/DistUpgradeView.py:408
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "시스템에 업그레이드할 것이 없습니다. 업그레이드를 취소합니다."

#: ../DistUpgrade/DistUpgradeView.py:421
msgid "Reboot required"
msgstr "다시 시작해야합니다."

#: ../DistUpgrade/DistUpgradeView.py:422
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "업그레이드가 끝났으며 다시 시작해야 합니다. 지금 하시겠습니까?"

#: ../DistUpgrade/DistUpgradeViewText.py:101
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"버그를 /var/log/dist-upgrade/main.log 파일과 /var/log/dist-upgrade/apt.log파일을 보고서에 "
"첨부하여 보고해주십시오. 업그레이드를 취소합니다.\n"
"원본 소스 목록을 /etc/apt/sources.list.distUpgrade 파일에 저장했습니다."

#: ../DistUpgrade/DistUpgradeViewText.py:125
msgid "Aborting"
msgstr "중지하는 중"

#: ../DistUpgrade/DistUpgradeViewText.py:130
msgid "Demoted:\n"
msgstr "강등됨:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:137
msgid "To continue please press [ENTER]"
msgstr "계속 하시려면 [엔터] 키를 눌러 주십시오"

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:208
#: ../DistUpgrade/DistUpgradeViewText.py:215
msgid "Continue [yN] "
msgstr "계속 [yN] "

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:208
msgid "Details [d]"
msgstr "자세한 내용 [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:174
#: ../DistUpgrade/DistUpgradeViewText.py:218
msgid "y"
msgstr "y"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:177
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:180
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:185
#, python-format
msgid "No longer supported: %s\n"
msgstr "더 이상 지원하지 않음: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:190
#, python-format
msgid "Remove: %s\n"
msgstr "지우기: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:200
#, python-format
msgid "Install: %s\n"
msgstr "설치: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:205
#, python-format
msgid "Upgrade: %s\n"
msgstr "업그레이드: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:222
msgid "Continue [Yn] "
msgstr "계속하겠습니까? [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:243
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"업그레이드를 완료하시려면 다시 시작해야합니다.\n"
"'y'를 선택하시면 시스템이 다시 시작합니다."

#: ../DistUpgrade/GtkProgress.py:73
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "%(total)li개 중 %(current)li번째 파일을 %(speed)s/s의 속도로 받는 중"

#: ../DistUpgrade/GtkProgress.py:79
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "%(total)li개 중 %(current)li번째 파일을 다운로드 중"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "각 파일의 진행 상태 표시"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "업그레이드 취소(_C)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "업그레이드 계속(_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>실행 중인 업그레이드를 취소하시겠습니까?</big></b>\n"
"\n"
"업그레이드를 취소하면 시스템을 사용할 수 없을 가능성이 있습니다.업그레이드를 계속 하는 것을 강력히 추천합니다."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "업그레이드 시작(_S)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "바꾸기(_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "파일 간의 차이점"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "버그 보고(_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "계속(_C)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>업그레이드를 시작하시겠습니까?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>시스템을 다시 시작 하면 업데이트를 완료합니다.</big></b>\n"
"\n"
"계속하기 전에 작업을 저장해 주십시오."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "배포판 업그레이드"

#: ../data/gtkbuilder/DistUpgrade.ui.h:19
msgid "<b><big>Upgrading Ubuntu to version 13.04</big></b>"
msgstr "<b><big>우분투를 13.04 버전으로 업그레이드</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "새 소프트웨어 채널을 설정하는 중"

#: ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "시스템 다시 시작"

#: ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "터미널"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "업그레이드(_U)"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>새 버전의 우분투를 사용할 수 있습니다. 업그레이드 하시겠습니까?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "업그레이드하지 않음"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "나중에 묻기"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "지금 업그레이드"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "새 우분투로 업그레이드 하는 것을 취소했습니다."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr "이후 언제라도 소프트웨어 업데이트 도구를 연 후 \"업그레이드\"를 선택해 업그레이드할 수 있습니다."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "배포판 업그레이드 실행"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "우분투를 업그레이드하려면 인증을 해주십시오."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "부분 업그레이드 실행"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "부분 업그레이드를 수행하려면 인증을 해주십시오."

#: ../do-partial-upgrade:67 ../do-release-upgrade:69
msgid "Show version and exit"
msgstr "버전을 표시하고 끝내기"

#: ../do-partial-upgrade:70 ../do-release-upgrade:76
msgid "Directory that contains the data files"
msgstr "데이터 파일이 저장된 디렉터리"

#: ../do-partial-upgrade:73 ../do-release-upgrade:89
msgid "Run the specified frontend"
msgstr "지정한 프론트엔드를 실행"

#: ../do-partial-upgrade:90
msgid "Running partial upgrade"
msgstr "부분 업그레이드 실행"

#: ../do-release-upgrade:31
msgid "Downloading the release upgrade tool"
msgstr "배포판 업그레이드 도구 다운로드 중"

#: ../do-release-upgrade:72 ../check-new-release-gtk:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr "최신 개발 버전으로 업그레이드할 수 있는지 확인합니다"

#: ../do-release-upgrade:79 ../check-new-release-gtk:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr "업그레이드 도구를 이용하여 $distro-proposed에서 최신 배포판으로 업그레이드를 시도합니다"

#: ../do-release-upgrade:83
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"특별 업그레이드 모드에서 실행합니다.\n"
"현재는 데스크톱 시스템의 일반적인 업그레이드를 위한 'desktop' 모드와 서버 시스템을 위한 'server' 모드를 지원합니다."

#: ../do-release-upgrade:91
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "샌드박스 aufs 오버레이로 시험 업그레이드"

#: ../do-release-upgrade:94
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr "새 배포판이 있을 경우에만 확인하기. 결과는 끝내기 코드를 통해 반환"

#: ../do-release-upgrade:108
msgid "Checking for a new Ubuntu release"
msgstr "새 우분투 배포판 확인"

#: ../do-release-upgrade:121
msgid "Your Ubuntu release is not supported anymore."
msgstr "현재 사용 중인 우분투 배포판은 더 이상 지원하지 않습니다."

#: ../do-release-upgrade:122
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"업그레이드 정보를 확인하려면, 아래 주소를 방문하세요:\n"
"%(url)s\n"

#: ../do-release-upgrade:128
msgid "No new release found"
msgstr "새 배포판이 없습니다."

#: ../do-release-upgrade:133
msgid "Release upgrade not possible right now"
msgstr "배포판을 업그레이드할 수 없습니다."

#: ../do-release-upgrade:134
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr "지금 배포판을 업그레이드할 수 없습니다. 나중에 다시 시도해주십시오.서버의 보고: '%s'"

#: ../do-release-upgrade:140
#, c-format
msgid "New release '%s' available."
msgstr "새 배포판 '%s'을(를) 사용할 수 있습니다."

#: ../do-release-upgrade:141
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "업그레이드를 하시려면 'do-release-upgrade' 명령을 실행하세요."

#: ../check-new-release-gtk:113
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "우분투 %(version)s(으)로 업그레이드할 수 있습니다."

#: ../check-new-release-gtk:144
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "우분투 %s(으)로 업그레이드하는 것을 취소했습니다."

#: ../check-new-release-gtk:186
msgid "Add debug output"
msgstr "디버그 출력 추가"

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr "부분 업그레이드를 실행하려면 인증이 필요합니다."

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "배포판을 업그레이드하려면 인증이 필요합니다."
